File: src/main/scala/im/tox/toktok/app/MessageActivity/MessageAttachments.scala
package im.tox.toktok.app.MessageActivity

import android.os.Bundle
import android.support.v4.app.Fragment
import android.view.{LayoutInflater, View, ViewGroup}
import im.tox.toktok.R


class MessageAttachments extends Fragment {

  override def onCreateView(inflater: LayoutInflater, container: ViewGroup, savedInstanceState: Bundle): View = {
    val view: View = inflater.inflate(R.layout.overlay_attachments, container, false);

    return view
  }

}

File: src/main/java/im/tox/toktok/app/message_activity/MessageAttachments.java
package im.tox.toktok.app.message_activity;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.ViewGroup;
import android.widget.LinearLayout;

import im.tox.toktok.R;

public final class MessageAttachments extends Fragment {

  @Override
  public LinearLayout onCreateView(@NonNull LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
    return (LinearLayout) inflater.inflate(R.layout.overlay_attachments, container, false);
  }

}
